src/Compilers/CSharp/Portable/Binder/Semantics/Conversions/ConversionsBase.cs

```diff
@@ -616,6 +616,7 @@ private static bool IsStandardImplicitConversionFromType(ConversionKind kind)
                 case ConversionKind.Identity:
                 case ConversionKind.ImplicitNumeric:
                 case ConversionKind.ImplicitNullable:
+                case ConversionKind.NullLiteral:
```

Comment:
>case ConversionKind.NullLiteral: [](http://example.com/codeflow?start=16&length=32)

This feels wrong. Null literal conversion is not a conversion from type. It is a conversion from expression.

Reply:
It looks like the helper above should be adjusted instead.